mycroft/tts/ivona_tts.py

```diff
@@ -0,0 +1,41 @@
+from mycroft.configuration import Configuration
+from mycroft.tts import TTS, TTSValidator
+import pyvona
+
+
+class Ivona(TTS):
+    def __init__(self, lang, config):
+        super(Ivona, self).__init__(lang, config, IvonaValidator(self))
+        self.config = Configuration.get().get("tts", {}).get("ivona", {})
+        self.access_key = self.config.get("access_key")
+        self.secret_key = self.config.get("secret_key")
+        self.gender = self.config.get("gender")
+        self.engine = pyvona.create_voice(self.access_key, self.secret_key)
+        if self.gender == 'female':
+            self.engine.voice_name = 'Salli'
+        else:
+            self.engine.voice_name = 'Joey'
+
+    def execute(self, sentence, ident=None):
```

Comment:
Looking at pyvona, it seems that it requires an extra dependency, `pygame`, if you use `engine.speak()`. Instead, why don't you override Mycroft TTS's [`get_tts()`](https://github.com/bhaveshAn/mycroft-core/blob/dab45c026850a9d3dfe8a25dcecea71fbf9bdeb4/mycroft/tts/mimic_tts.py#L141) method and use `engine.fetch_voice('text', '/file/name')`? Using `fetch_voice` doesn't require the extra dependency.